packages/twenty-front/src/modules/activities/files/components/AttachmentList.tsx

```diff
@@ -17,7 +26,8 @@ const StyledContainer = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: center;
-  padding: 8px 24px;
+  padding: 8px 24px 24px;
```

Comment:
We should use `theme.spacing` here:

```suggestion
  padding: ${({ theme }) => theme.spacing(2, 6, 6)};
```